### PR TITLE
Feature/rename events

### PR DESCRIPTION
**Pull request by ThomazFB**
Rename the current events to better suit the purposed description text

### CODE REVIEW

File: clappr/src/main/kotlin/io/clappr/player/base/Events.kt [hunk 1]
```diff
@@ -39,19 +39,19 @@ enum class Event(val value: String) {
      */
     DID_SEEK("didSeek"),
     /**
-     * Media source changed
+     * Media source load
```
Comment (fcapinho) [inline clappr/src/main/kotlin/io/clappr/player/base/Events.kt]: Change to "loaded"

Comment (ThomazFB) [reply clappr/src/main/kotlin/io/clappr/player/base/Events.kt]: done

File: clappr/src/main/kotlin/io/clappr/player/base/Events.kt [hunk 2]
```diff
@@ -39,19 +39,19 @@ enum class Event(val value: String) {
...
      */
-    DID_CHANGE_SOURCE("didChangeSource"),
+    DID_LOAD_SOURCE("didLoadSource"),
 
 
     /**
      * Media buffer percentage updated
      */
-    BUFFER_UPDATE("bufferUpdate"),
+    DID_UPDATE_BUFFER("bufferUpdate"),
```
Comment (pmunizferreira) [inline clappr/src/main/kotlin/io/clappr/player/base/Events.kt]: What about maintain value with same name of event, but camel case?

File: clappr/src/main/kotlin/io/clappr/player/base/Events.kt [hunk 3]
```diff
@@ -27,9 +27,9 @@ enum class Event(val value: String) {
      */
     DID_PAUSE("didPause"),
     /**
-     * Changed to STALLED state
+     * Changed to STALLING state
```
Comment (pmunizferreira) [inline clappr/src/main/kotlin/io/clappr/player/base/Events.kt]: I suggest creating class `EventTest` with 

```

    @Test
    fun shouldHaveUniqueValue() {
        Event.values().forEach {
            assertEquals(1, Event.values().filter { event -> event.value == it.value  }.size, "More than 1 event with \"${it.value}\" value")
        }
    }
```

Comment (ThomazFB) [reply clappr/src/main/kotlin/io/clappr/player/base/Events.kt]: done

File: clappr/src/main/kotlin/io/clappr/player/base/Events.kt (hunk 2 above)
Comment (ThomazFB) [reply clappr/src/main/kotlin/io/clappr/player/base/Events.kt]: done

**Pull request merged**